community/modules/scheduler/htcondor-configure/README.md

```diff
@@ -72,9 +72,81 @@ example can be found in the [examples README][htc-example].
   - internal_ip
 ```
 
-A full example can be found in the [examples README][htc-example].
+## High Availability
 
-[htc-example]: ../../../../examples/README.md#htcondor-poolyaml--
+This module supports high availability modes of the HTCondor Central Manager and
+of the Access Points. In these modes, the services can be resiliant against
+zonal failures by distributing the services across two zones. To implement
+HA of the Central Manager service, apply the startup script in the example
+snipppet to 2 VMs. The 2 VMs should differ only by using the primary and
+secondary IP addresses created by this module.
```

Comment:
Subnetwork is also different between the two `vm-instances` defined below. Not sure if this is a typo or correct but either way it seems to conflict with the way the sentence is written: `should differ only by`.

Reply:
I've updated both the blueprint setting and surrounding text.